### PR TITLE
Characters inside HTML tags <code>...</code> are not escaped.

### PR DESCRIPTION
**Pull request by vonavi**
Hey!

Org-mode snippet `=http://<app>-<namespace>.rhcloud.com=` is converted as

``` html
<code>http://<app>-<namespace>.rhcloud.com</code>
```

whereas it should be

``` html
<code>http://&lt;app&gt;-&lt;namespace&gt;.rhcloud.com</code>
```

Best regards,
Vladimir.

### CODE REVIEW

**Comment by vonavi** [comment]
Fix done.

File: lib/org-ruby/html_output_buffer.rb [hunk 1]
```diff
@@ -229,21 +232,28 @@ def mode_is_table?(mode)
...
         if $`[-2..-1] == "@<" then $&
         else "#{$1}&gt;"
         end
       end
-      str.gsub!(/@(<[^<>\n]*>)/, "\\1")
+      @buffer.gsub!(/@(<[^<>\n]*>)/, "\\1")
+    end
+
+    # Escapes any HTML content in string beyond @buffer.
+    def escape_html! str
```
Comment (wallyqs) [inline lib/org-ruby/html_output_buffer.rb]: I think we could switch this one (#253) with Ruby's native CGI::escape_html(str)
http://www.ruby-doc.org/stdlib-1.9.2/libdoc/cgi/rdoc/CGI.html#method-c-escape_html

**Comment by vonavi** [comment]
Hey, Wally!
Thanks for the notice. I've done it. It escapes quotes `"` in addition.
Best regards,
Vladimir.

**Comment by wallyqs** [comment]
Merged, thanks again :)

**Pull request merged**